src/components/ConditionalFilter/API/ProductFilterAPIProvider.tsx

```diff
@@ -79,8 +85,31 @@ export const useProductFilterAPIProvider = (): FilterAPIProvider => {
     return handler.fetch();
   };
 
+  const useInitialState = (fetchingParams: FetchingParams) => {
```

Comment:
suggestion: `useInitialState` sounds like hook but this seems to be more like normal method/function

Reply:
it calls hook inside (useDataFromAPI) which calls react hooks